pubspec.yaml

```diff
@@ -31,6 +31,7 @@ flutter:
 
   assets:
     - images/
+    - data/
```

Comment:
Maybe you can renamed to mock, do you think that is more appropiated name?

Reply:
makes sense to me, changed!